src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -122,6 +147,31 @@ void GcsServer::Start() {
   gcs_init_data->AsyncLoad([this, gcs_init_data] { DoStart(*gcs_init_data); });
 }
 
+void GcsServer::CacheAndSetClusterId() {
+  static std::string const kTokenNamespace = "cluster";
```

Comment:
Can we define both constants (kTokenNamespace, kClusterIdKey) in the same place?

Reply:
Right now nobody else uses kTokenNamespace, though